docs/content/Schema/Fundamentals/Working-with-Joins.mdx

```diff
@@ -517,6 +531,72 @@ cube('Customers', {
 });
 ```
 
+We can then query the cube as follows:
+
+```json
+{
+  "dimensions": ["Users.company"],
+  "timeDimensions": [
+    {
+      "dimension": "Orders.createdAt"
+    }
+  ],
+  "filters": [
+    {
+      "member": "Orders.count",
+      "operator": "equals",
+      "values": ["0"]
+    }
+  ]
+}
+```
+
+### Using views
+
+Views can also be used in Cube to represent the previous two scenarios;
+
+```javascript
+view('TotalRevenuePerCustomer', {
+  description: `Total revenue per customer`,
+
+  measures: {
+    total: {
+      sql: `${Orders.totalRevenue}`,
+      // We specify the type here as `number` because we don't need to sum
+      // this data again
+      type: `number`,
+    },
+  },
+
+  dimensions: {
+    customer: {
+      sql: `${Users.company}`,
+      type: `string`,
+    },
+  },
```

Comment:
These can probably be `includes` instead, but there seems to be a bug in Cube preventing me from testing this right now